### PR TITLE
[T3A3][W13-A4]BernardKoh

### PR DESCRIPTION
**Pull request by bernardified**

### CODE REVIEW

**Comment by bernardified** [comment]
Ready for review.

File: doc/UserGuide.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
+* `add John Doe p/98765432 e/johnd@gmail.com a/123, John Street, #01-01, 123456`
+* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/123, Newgate Prison, #01-01, 123456 t/criminal t/friend`
```
Comment (damithc) [inline doc/UserGuide.md]: Good job updating user guide.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -8,13 +8,20 @@
...
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, #12-34, 123456";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in BLOCK, STREET, UNIT,"
+    															+ " POSTAL_CODE";
+    
+    public static final String ADDRESS_VALIDATION_REGEX = ".+, .+, .+, .+";
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java]: Good to see attempting to use regex

File: src/seedu/addressbook/data/person/Block.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Block.java]: Missing header comment. All non trivial classes/methods should have javadoc format header comments.

**Comment by damithc** [comment]
Very good effort @bernardified 
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**